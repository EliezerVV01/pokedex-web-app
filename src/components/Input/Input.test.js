import React from 'react';
import Input from './Input';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Input  invalid={true}
    touched={true}
    value={'Eating'}
    changed={()=>alert('It changed')}></Input>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});