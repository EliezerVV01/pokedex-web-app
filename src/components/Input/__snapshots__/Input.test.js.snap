// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Input"
>
  <label
    className="Label"
  />
  <input
    className="InputElement Invalid"
    onChange={[Function]}
    value="Eating"
  />
</div>
`;
